thisfinans/src/main/resources/application.yml

```
@@ -1,7 +1,7 @@
 spring:
```

adiciona  -  hibernate: ddl-auto: validate

https://www.baeldung.com/spring-boot-data-sql-and-schema-sql

```suggestion
spring:
  datasource:
    url: jdbc:mysql://localhost:3306/thisfinansdb
    username: root
    password: 1234567
  jpa:
    open-in-view: false
    hibernate:
      ddl-auto: validate
```